docs/upgrade.html

```diff
@@ -21,10 +21,15 @@
 
 <h5><a id="upgrade_250_notable" href="#upgrade_250_notable">Notable changes in 2.5.0</a></h5>
 <ul>
+    <li>When <code>RebalanceProtocol#COOPERATIVE</code> is used, <code>Consumer#poll</code> can still return data
+        while it is in the middle of a rebalance for those partitions still owned by the consumer; in addition
+        <code>Consumer#commitSync</code> now may throw a non-fatal <code>RebalanceInProgressException</code> to notify
+        users of such an event, in order to distinguish from the fatal <code>CommitFailuedException</code> and allow
+        users to complete the ongoing rebalance and then reconsider committing offsets.</li>
```

Comment:
```suggestion
        users to complete the ongoing rebalance and then reattempt committing offsets for those still-owned partitions.</li>
```
This one is just a proposal, feel free to take it or leave it. We could also maybe link to the new Javadocs for more details, "just in case" users don't read all the docs/changes carefully before jumping on this